versions.lock

```diff
@@ -268,11 +273,15 @@ org.glassfish.jersey.inject:jersey-hk2:2.35 (1 constraints: de04fe30)
 org.glassfish.jersey.media:jersey-media-json-jackson:2.35 (1 constraints: de04fe30)
 org.hamcrest:hamcrest:2.2 (3 constraints: 7620ce25)
 org.hamcrest:hamcrest-core:2.2 (1 constraints: cc05fe3f)
+org.immutables:value-annotations:2.9.2 (1 constraints: 0f051236)
 org.itadaki:bzip2:0.9.1 (2 constraints: bd0c4b2c)
 org.javassist:javassist:3.25.0-GA (1 constraints: 2a110ef1)
 org.jctools:jctools-core:3.3.0 (1 constraints: 08050336)
 org.jdom:jdom2:2.0.6.1 (1 constraints: be0c371b)
 org.junit:junit-bom:5.9.0 (1 constraints: c7116dde)
+org.locationtech.jts:jts-core:1.19.0 (2 constraints: a31de760)
```

Comment:
Woah; we're adding a dependency on JTS?  Okay; there's a Solr JIRA that debates doing this for Solr-core.
I wonder if it's actually used, though?  Like is this for features that a Calcite/Solr user is unlikely to use?  I'm not sure honestly.

Reply:
JTS was added to support a bunch of functions in Calcite - that we should get for free.

* https://calcite.apache.org/news/2022/09/10/release-1.32.0/
* https://calcite.apache.org/docs/history.html#v1-32-0
* https://issues.apache.org/jira/browse/CALCITE-4294
* https://issues.apache.org/jira/browse/CALCITE-5262

Reply:
This replaces `com.esri.geometry:esri-geometry-api:2.2.0 (1 constraints: 5c0db22c)`

https://github.com/apache/solr/pull/1031/commits/fafaaa0fd33f70b535a9fdd947e5742af21201d4#diff-14184d8ebd3edfbb9a6a4454c522968d3ade7e606339876f1fa50f768ea8e897L8

Reply:
@joel-bernstein do you gather the above features in Calcite would actually be useful to the way Solr integrates with Calcite, or not, and thus this is just unwelcome dependency bloat?  I'm pushing us to consciously think of these things when upgrading libraries that bring in transitive dependencies.

Reply:
I'm extremely against removing dependencies all willy nilly just to meet some weird  goal of "dependency bloat" - yes JTS is needed in Calcite - it enables SQL support for geo functions. Just like esri did. The whole work of moving SQL out of core was so that we could have a thinner core - while still maintaining capabilities.

I saw you asked on Calcite if JTS is optional - it wouldn't be in the Maven dependency tree.